merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PreparedStatements.java

```diff
@@ -29,6 +31,13 @@ private PreparedStatements() {}
           .appendOffset("+HH:mm:ss", "+00")
           .toFormatter();
 
+  private static final @Language("SQL") String intervalStylePrefix = "set intervalstyle = ";
+
+  public static void setIntervalStyle(final Connection connection, final String intervalStyle) throws SQLException {
```

Comment:
This is very minor but I would suggest either renaming this something like `setIntervalStyleISO8601` and removing the `String` param. or making `intervalStyle` an enum with currently one option, `iso_8601`.

Reply:
There's four options, but yeah for now we're only using iso8601. I'll make an enum.

Reply:
Thanks :) Feel free to resolve this when done